### ISSUE TITLE
my portal cannot connect to the blockstack core node

### ISSUE THREAD

**Issue by shea256** [blockstack/blockstack-browser#389]
Here are the logs:

```
default	15:05:13.130226 -0400	Blockstack	Traceback (most recent call last):
  File "/tmp/blockstack-venv/bin/blockstack", line 45, in <module>
    from blockstack_client.cli import run_cli, exit_with_error
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/__init__.py", line 24, in <module>
default	15:05:13.130365 -0400	Blockstack	    import app
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/app.py", line 40, in <module>
    import wallet
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/wallet.py", line 50, in <module>
    from .backend.crypto.utils import get_address_from_privkey
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/__init__.py", line 21, in <module>
    import registrar
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/registrar.py", line 49, in <module>
    from .queue import get_queue_state, in_queue, queue_removeall
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/queue.py", line 33, in <module>
    from ..proxy import get_default_proxy
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/proxy.py", line 42, in <module>
    import storage
  File "/private/tmp/blockstack-venv/lib/python2.7/site-pac
default	15:05:13.151689 -0400	Blockstack	Finished Blockstack Core setup!
default	15:05:13.151774 -0400	Blockstack	Starting Blockstack Core API endpoint...
default	15:05:13.565187 -0400	Blockstack	Traceback (most recent call last):
  File "/tmp/blockstack-venv/bin/blockstack", line 45, in <module>
    from blockstack_client.cli import run_cli, exit_with_error
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/__init__.py", line 24, in <module>
    import app
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/app.py", line 40, in <module>
default	15:05:13.565318 -0400	Blockstack	    import wallet
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/wallet.py", line 50, in <module>
    from .backend.crypto.utils import get_address_from_privkey
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/__init__.py", line 21, in <module>
    import registrar
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/registrar.py", line 49, in <module>
    from .queue import get_queue_state, in_queue, queue_removeall
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/queue.py", line 33, in <module>
    from ..proxy import get_default_proxy
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/proxy.py", line 42, in <module>
    import storage
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/storage.py", line 47, in <module>
    from scripts import is_name_valid
  File "/private/tmp/blockstack-
default	15:05:13.565405 -0400	Blockstack	    from fastecdsa import curvemath
ImportError: dlopen(/private/tmp/blockstack-venv/lib/python2.7/site-packages/fastecdsa/curvemath.so, 2): Library not loaded: /usr/local/opt/gmp/lib/libgmp.10.dylib
  Referenced from: /private/tmp/blockstack-venv/lib/python2.7/site-packages/fastecdsa/curvemath.so
  Reason: image not found
default	15:05:13.586956 -0400	Blockstack	Blockstack Core API started!
default	15:05:41.397888 -0400	Blockstack	statusItemClick
default	15:05:42.602465 -0400	Blockstack	exitClick
default	15:05:43.976647 -0400	Blockstack	User decided to exit...
default	15:05:43.978904 -0400	Blockstack	Blockstack Portal proxy terminated
default	15:05:43.978986 -0400	Blockstack	CORS proxy terminated
default	15:05:43.979055 -0400	Blockstack	Attempting to stop Blockstack Core API...
default	15:05:44.358345 -0400	Blockstack	Traceback (most recent call last):
  File "/tmp/blockstack-venv/bin/blockstack", line 45, in <module>
    from blockstack_client.cli import run_cli, exit_with_error
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/__init__.py", line 24, in <module>
    import app
default	15:05:44.358458 -0400	Blockstack	  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/app.py", line 40, in <module>
    import wallet
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/wallet.py", line 50, in <module>
    from .backend.crypto.utils import get_address_from_privkey
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/__init__.py", line 21, in <module>
    import registrar
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/registrar.py", line 49, in <module>
    from .queue import get_queue_state, in_queue, queue_removeall
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/backend/queue.py", line 33, in <module>
    from ..proxy import get_default_proxy
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstack_client/proxy.py", line 42, in <module>
    import storage
  File "/private/tmp/blockstack-venv/lib/python2.7/site-packages/blockstac
default	15:05:44.358539 -0400	Blockstack	ImportError: dlopen(/private/tmp/blockstack-venv/lib/python2.7/site-packages/fastecdsa/curvemath.so, 2): Library not loaded: /usr/local/opt/gmp/lib/libgmp.10.dylib
  Referenced from: /private/tmp/blockstack-venv/lib/python2.7/site-packages/fastecdsa/curvemath.so
  Reason: image not found
default	15:05:44.379418 -0400	Blockstack	Blockstack Core api stopped.
default	15:05:44.379565 -0400	Blockstack	Goodbye!
```

**Comment by shea256** [comment]
Another user had this same problem: https://github.com/blockstack/blockstack-portal/issues/388

**Comment by larrysalibra** [comment]
It looks like `fastecdsa` linked to my `brew` installation of GMP instead of the one in the `virtualenv` - not sure why it ignored the linker flags.

I'll see if I can fix it and push a new build.

**Comment by larrysalibra** [comment]
Should be fixed in 0.9.0. Let me know if you're still having issues!
